paasta_tools/spark_tools.py

```diff
@@ -95,13 +101,12 @@ def get_default_event_log_dir(**kwargs) -> str:
         spark_run_conf = {}
 
     try:
-        account_id = (
```

Comment:
This is referenced below, you shouldn't just return here

Reply:
yeah this mistake was caught in the tests and my next update will fix it